Airports/EGLL/Stars.txt

```diff
@@ -14,42 +14,27 @@ STAR:EGLL:27R:BNN4A:HON TOBID SOPIT WCO BNN FI27R RW27R
 STAR:EGLL:27L:BNN4A:HON TOBID SOPIT WCO BNN FI27L RW27L
 STAR:EGLL:09R:BNN4A:HON TOBID SOPIT WCO BNN FI09R RW09R
 STAR:EGLL:09L:BNN4A:HON TOBID SOPIT WCO BNN FI09L RW09L
+STAR:EGLL:27R:HAZEL1H:HAZEL LLS01 OCK FI27R RW27R
+STAR:EGLL:27L:HAZEL1H:HAZEL LLS01 OCK FI27L RW27L
+STAR:EGLL:09R:HAZEL1H:HAZEL LLS01 OCK FI09R RW09R
+STAR:EGLL:09L:HAZEL1H:HAZEL LLS01 OCK FI09L RW09L
 STAR:EGLL:27R:LOGAN1H:LOGAN TRIPO SABER BRASO LAM FI27R RW27R
 STAR:EGLL:27L:LOGAN1H:LOGAN TRIPO SABER BRASO LAM FI27L RW27L
 STAR:EGLL:09R:LOGAN1H:LOGAN TRIPO SABER BRASO LAM FI09R RW09R
 STAR:EGLL:09L:LOGAN1H:LOGAN TRIPO SABER BRASO LAM FI09L RW09L
-STAR:EGLL:27R:OCK1A:KENET OCK FI27R RW27R
-STAR:EGLL:27L:OCK1A:KENET OCK FI27L RW27L
-STAR:EGLL:09R:OCK1A:KENET OCK FI09R RW09R
-STAR:EGLL:09L:OCK1A:KENET OCK FI09L RW09L
-STAR:EGLL:27R:OCK1D:HON WOD OCK FI27R RW27R
-STAR:EGLL:27L:OCK1D:HON WOD OCK FI27L RW27L
-STAR:EGLL:09R:OCK1D:HON WOD OCK FI09R RW09R
-STAR:EGLL:09L:OCK1D:HON WOD OCK FI09L RW09L
-STAR:EGLL:27R:OCK2C:SAM HAZEL OCK FI27R RW27R
-STAR:EGLL:27L:OCK2C:SAM HAZEL OCK FI27L RW27L
-STAR:EGLL:09R:OCK2C:SAM HAZEL OCK FI09R RW09R
-STAR:EGLL:09L:OCK2C:SAM HAZEL OCK FI09L RW09L
-STAR:EGLL:27R:OCK2F:BEDEK NIGIT OCK FI27R RW27R
-STAR:EGLL:27L:OCK2F:BEDEK NIGIT OCK FI27L RW27L
-STAR:EGLL:09R:OCK2F:BEDEK NIGIT OCK FI09R RW09R
-STAR:EGLL:09L:OCK2F:BEDEK NIGIT OCK FI09L RW09L
-STAR:EGLL:27R:OCK3E:BILNI KUMIL ELDER HAZEL OCK FI27R RW27R
-STAR:EGLL:27L:OCK3E:BILNI KUMIL ELDER HAZEL OCK FI27L RW27L
-STAR:EGLL:09R:OCK3E:BILNI KUMIL ELDER HAZEL OCK FI09R RW09R
-STAR:EGLL:09L:OCK3E:BILNI KUMIL ELDER HAZEL OCK FI09L RW09L
-STAR:EGLL:27R:OCK4B:DOMUT KATHY HAZEL OCK FI27R RW27R
-STAR:EGLL:27L:OCK4B:DOMUT KATHY HAZEL OCK FI27L RW27L
-STAR:EGLL:09R:OCK4B:DOMUT KATHY HAZEL OCK FI09R RW09R
-STAR:EGLL:09L:OCK4B:DOMUT KATHY HAZEL OCK FI09L RW09L
-STAR:EGLL:27R:OTMET1H:OTMET SOKDU NEDUL BEGTO HAZEL LLS01 OCK FI27R RW27R
-STAR:EGLL:27L:OTMET1H:OTMET SOKDU NEDUL BEGTO HAZEL LLS01 OCK FI27L RW27L
-STAR:EGLL:09R:OTMET1H:OTMET SOKDU NEDUL BEGTO HAZEL LLS01 OCK FI09R RW09R
-STAR:EGLL:09L:OTMET1H:OTMET SOKDU NEDUL BEGTO HAZEL LLS01 OCK FI09L RW09L
```

Comment:
Are the OTMET STARs supposed to be deleted?

Reply:
Not that im aware off, may of mistaken for OCK1H will add back